sdk/spring-sdk/src/main/scala/kalix/springsdk/KalixClient.scala

```diff
@@ -76,6 +76,32 @@ trait KalixClient {
    */
   def post[P, R](uri: String, body: P, returnType: Class[R]): DeferredCall[Any, R]
 
+  /**
+   * Provides utility to do a POST HTTP request to a target endpoint belonging to a Kalix component. Such endpoint is
+   * identified by a resource path (i.e. excluding authority and scheme).
+   *
+   * Example of use:
+   * {{{
+   *     public Effect<Confirmation> createCounter() {
+   *       var serviceCall = kalixClient.post("/counter/create", Confirmation.class);
+   *       return effects().forward(serviceCall);
+   *     }
+   * }}}
+   *
+   * @param uri
+   *   The resource path where the target endpoint will be reached at. Query parameters can be passed in as part of the
+   *   URI but should be encoded if containing special characters.
+   * @param body
+   *   The HTTP body type expected by the target endpoint
```

Comment:
```suggestion
```

Reply:
Of course, I was only paying attention to the P param. 🤦